README.adoc

```diff
@@ -268,14 +304,14 @@ In the [hotspot=import file=0]`import` statements of this test class, you'll not
 test has some new dependencies. Before the test can be compiled by Maven, you need to update 
 the [hotspot file=1]`pom.xml` to include these dependencies.
 
-The Apache [hotspot=commons-httpclient file=1]`commons-httpclient` and [hotspot=junit file=1]`junit` 
+The Apache [hotspot=commons-httpclient file=1]`commons-httpclient` and [hotspot=junit file=1]`junit-jupiter-engine` 
 dependencies are needed to compile and run the integration test [hotspot=EndpointIT file=0]`EndpointIT` 
 class. The scope for each of the dependencies is set to [hotspot=test1 hotspot=test2 file=1]`test` because 
 the libraries are needed only during the Maven build and do not needed to be packaged with 
 the application.
 
-Now, during the Maven build, Maven creates the WAR file that contains the web application 
-and then runs any integration test classes (classes with names that end in `IT`) that it 
+Now, the created WAR file contains the web application 
+and then development mode can run any integration test classes (classes with names that end in `IT`) that it 
```

Comment:
Change
"Now, the created WAR file contains the web application and then development mode can run any integration test classes (classes with names that end in `IT`) that it finds."
To
"Now, the created WAR file contains the web application, and development mode can run any integration test classes that it finds. Integration test classes are classes with names that end in `IT`."